screens/main/recordedWalks.js

```diff
@@ -10,16 +10,14 @@ import {
   Image,
   Platform,
 } from 'react-native';
-import {Button, InfoBox} from '../../components';
+import {Button, InfoBox, PageTitle} from '../../components';
 import {Colors, GlobalStyles} from '../../styles';
 
 export default function RecordedWalksScreen({navigation}) {
   return (
     <SafeAreaView style={styles.container}>
       <ScrollView>
-        <View style={styles.content}>
-          <Text style={GlobalStyles.h1}>Recorded Walks</Text>
-        </View>
+        <PageTitle title={"My Recorded Walks"} />
```

Comment:
I think this can just be `title="My Recorded Walks"`

Reply:
Fixed locally.